lib/PayPal/Api/FuturePayment.php

```diff
@@ -32,7 +32,7 @@ public function create($apiContext = null, $clientMetadataId = null)
             );
         }
         $payLoad = $this->toJSON();
-        $call = new PayPalRestCall($apiContext);
+        $call = $call ? $call : new PayPalRestCall($apiContext);
```

Comment:
Curious, if we could just replace this with `executeCall` itself. Maybe we can do that as a separate PR.